### PR TITLE
addresses all issues

### PR DESCRIPTION
**Pull request by vicfriedman**
@jmburges

### CODE REVIEW

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    end
+
+    it "redirects to the recipe show page" do 
+      expect(last_request.url).to eq("http://example.org/recipes/#{Recipe.last.id}")
+    end
+  end
+
+  describe "updating a recipe" do
+    before do
+      @cookie = Recipe.create(
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: Why do you have these variables? the `@cookie` and `@cake`?

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+  end
+
+  describe "new page '/recipes/new'" do
+    before do
+      get "/recipes/new"
+    end
+
+    it 'responds with a 200 status code' do
+      # get "/recipes/new"
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: commented out code

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      post '/recipes', params
+      follow_redirect!
+    end
+    it "creates a new recipe and saves to the database" do 
+      expect(Recipe.all.count).to eq(3)
+      expect(Recipe.last.name).to eq("pumpkin pie")
+    end
+
+    it "redirects to the recipe show page" do 
+      expect(last_request.url).to eq("http://example.org/recipes/#{Recipe.last.id}")
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: Any way you can do this without the full url of `example.org` stuff and just have the relative path? I think the example.org is going to confuse them

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    end
+  end
+
+  describe "edit page '/recipes/:id/edit'" do
+    before do
+      get "/recipes/#{@recipe1.id}/edit"
+    end
+
+    it 'responds with a 200 status code' do
+      # get "/recipes/#{@recipe1.id}/edit"
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: commented out code

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    @recipe2 = Recipe.create(:name => "waldorf salad", :ingredients => "apples, cabbage, oil, vinegar", :cook_time => "0")
+  end
+
+  describe "Index page '/recipes'" do
+    before do
+      get "/recipes"
+    end
+
+    it 'responds with a 200 status code' do
+      # get '/recipes'
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: commented out code

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  end
+
+    
+  describe "show page '/recipes/:id'" do
+    before do
+      get "/recipes/#{@recipe1.id}"
+    end
+
+    it 'responds with a 200 status code' do
+      # get "/recipes/#{@recipe1.id}"
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: commented code

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 1 above)
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: ooooo I get the usage of these things, but why are you storing @cake into a variable?

File: spec/sinatra_restful_routes_lab_spec.rb [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+        :ingredients => "chocolate chips, flour, sugar, butter, cocoa powder",
+        :cook_time  => "30 minutes",
+      }
+      post "/recipes/#{@cookie.id}", params
+      follow_redirect!
+    end
+
+    it "deletes a recipe" do
+      post "/recipes/#{@cookie.id}/delete"
+      expect(Recipe.find_by_name(@cookie.name)).to eq(nil)
```
Comment (jmburges) [inline spec/sinatra_restful_routes_lab_spec.rb]: find_by_id not find_by_name

**Comment by jmburges** [comment]
bunch of small issues and then :shipit:

**Comment by jmburges** [comment]
and by :shipit: I mean merge, and mark complete on the spreadsheet

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 1 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: removed `@cake`

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 2 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: remvoed

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 3 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: done

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 4 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: removed

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 5 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: removed

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 6 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: removed

File: spec/sinatra_restful_routes_lab_spec.rb (hunk 7 above)
Comment (vicfriedman) [inline spec/sinatra_restful_routes_lab_spec.rb]: done

**Pull request merged**